### DISCUSSION TITLE
Experience Portal error on asset download

### DISCUSSION THREAD

**Discussion by abrar123alhaq** [pimcore/pimcore#13631] [Q&A / Support]
Hello,

I am facing an issue when downloading image asset with from experience portal.
![image](https://user-images.githubusercontent.com/832552/203023578-bf46cb33-cb34-464a-90ee-3acb9aad550c.png)

Steps to reproduce error:
1. Download asset and select Custom option
![image](https://user-images.githubusercontent.com/832552/203023766-c4ff3fe1-4261-4624-a216-59ee705c07cd.png)
2. Do not set Width and height click download button.
3. Now getting "width" array key Undefined

I will be grateful if anyone could help.

**Comment by kingjia90** [comment]
Thank you for the report!

I've opened an issue on the right repository https://github.com/pimcore/portal-engine/issues/281 and already provided a [PR ](https://github.com/pimcore/portal-engine/pull/282) that will fix this
If you don't have access to it, please contact [enterprise-support@pimcore.com](mailto:enterprise-support@pimcore.com).

**Reply by abrar123alhaq** [reply]
Thank you for the answer.
I am getting 404 page not found error when trying to access PR https://github.com/pimcore/portal-engine/pull/282

**Reply by kingjia90** [reply]
it"s because is a private repo, you have to be granted access by contacting that email
